content/en/docs/collector/build-connector.md

```diff
@@ -0,0 +1,567 @@
+---
+title: Building a Connector
+spelling:
+  cSpell:ignore spanmetrics servicegraph exampleconnector struct Errorf
+  cSpell:ignore mapstructure pdata mapstructure pmetric ptrace uber gord
+  cSpell:ignore gomod loggingexporter batchprocessor otlpreceiver
+---
+
+## Connectors in OpenTelemetry
+
+The content of this page is most applicable if you already have an instrumented
+application generating some kind of tracing telemetry data and already have an
+understanding of the [OpenTelemetry Collector](/docs/collector).
+
+## What is a Connector?
+
+Each pipeline in the OpenTelemetry Collector acts on one type of telemetry data.
+There may exist the need to process one form of telemetry data into another one,
+but it is required to route the according data to its proper collector pipeline.
+A connector acts as the means of sending telemetry data between different
+collector pipelines by connecting them. A connector acts as an exporter to one
+pipeline and a receiver to another.
```

Comment:
The last two sentences of this section are a good description of connectors in general. However, the section opens with a description of a subset of connectors - those that derive one type of data from another.

Since this is introducing connectors in general, do you think it makes sense to lead this section with the general description first and then describe more specific capabilities?